.buildkite/0_webui.yml

```diff
@@ -0,0 +1,14 @@
+agents:
+  queue: "juliaecosystem"
+  sandbox.jl: true
+
+steps:
+  - label: ":rocket: Launch Pipeline!"
+    branches: "main release-* v*"
```

Comment:
BTW if you add this, this means that PRs won't run this.  We still want to run PRs, right?

Reply:
I'm not sure we do. Wouldn't that allow any random PR that gets opened to build *and* deploy the multi-documentation?

Reply:
Usually what I do is to split up the docs build and deploy into two separate steps; first the docs get built for everyone, and then we upload a tarball of them as a buildkite artifact.  Then I have a second job that depends on the first, which downloads them from buildkite, and then uploads them to S3.  We can switch to that once we're certain that this is working.  :)

Reply:
FWIW, the documentation of SciMLDocs itself (NOT the aggregated multi-documentation) gets built when PRs are opened.

> Usually what I do is to split up the docs build and deploy into two separate steps; first the docs get built for everyone, and then we upload a tarball of them as a buildkite artifact. Then I have a second job that depends on the first, which downloads them from buildkite, and then uploads them to S3. We can switch to that once we're certain that this is working. :)

I don't know if storing them as buildkite artifacts is something we should be doing since the aggregated multi-documentation is **massive**.
But yes, if required, we can do that once this is verified to be working.